ana_scans.py

```diff
@@ -125,15 +125,22 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
   try:
     log = file("%s/anaLog.log"%(dirPath),"w")
  
-    #runCommand(cmd,log)
-    runCommand(cmd)
+    #returncode = runCommand(cmd,log)
+    returncode = runCommand(cmd)
+    if returncode != 0:
+      print "Error: command exited with non-zero code %d" % returncode
+      return returncode
```

Comment:
What's the reason behind this?  This block is already inside a `try:` block so any error results in the code executing the `except` portion and issues an error message already.

Please revert.

Reply:
Non-zero return values and exceptions aren't the same thing.

Reply:
Sure but I think this should be a separate PR. Could you revert here and open a new issue that is related to this? That would allow us to be a bit more organized and focused in our PR's